### PR TITLE
Fix: UX enhancements

### LINKED ISSUE #103: UI/UX Enhancements
- [x] Add share icon in app-navbar for shared apps
- [x] Transform Back icon in app-navbar into a rounded-button
- [x] Fix Metrics stats limits in dashboard homepage
- [x] Better Error handling in Onboarding App creation - It renders a static error statement - We need to use server response to render the error message.

### PR DESCRIPTION
**Pull request by jyash97**
Fix #103

### CODE REVIEW

File: src/components/AppHeader/AppHeader.js [hunk 1]
```diff
@@ -47,11 +55,19 @@ const AppHeader = ({
 		<Menu mode="horizontal">
 			<Menu.Item key="back" className={noBorder} style={{ padding: 0 }}>
 				<Link to="/">
-					<Icon type="arrow-left" />
+					<Button css={backIcon} shape="circle" icon="arrow-left" />
 				</Link>
 			</Menu.Item>
 			<Menu.Item key="1" className={noBorder}>
-				<span>{currentApp || 'Loading...'}</span>
+				{appOwner
```
Comment (metagrover) [inline src/components/AppHeader/AppHeader.js]: How will this work with App Switcher?

image

Comment (jyash97) [reply src/components/AppHeader/AppHeader.js]: We can show the owner on hover and on click we can show apps list

Comment (metagrover) [reply src/components/AppHeader/AppHeader.js]: I feel it'd be better if we only show the tooltip on hover at the share icon.

Comment (jyash97) [reply src/components/AppHeader/AppHeader.js]: Yeah that can be done. Will fix this

**Pull request closed without merging**